spec/spec.md

```diff
@@ -349,6 +345,21 @@ https://github.com/trustoverip/tswg-keri-specification/issues/69
 https://github.com/trustoverip/tswg-keri-specification/issues/70
 :::
 
+[[def: Dead-Attack]]
+
+~ See (Security Properties of Prerotation)[#dead-attacks]
```

Comment:
```suggestion
~ an attack on an establishment event that occurs after the Key-state for that event has become stale because a later establishment event has rotated the sets of signing and pre-rotated keys to new sets. See (Security Properties of Prerotation)[#dead-attacks]
```